recipe/meta.yaml

```diff
@@ -39,7 +39,8 @@ test:
   imports:
     - siphash24
   commands:
-    - "PYTHONHASHSEED=0 python check.py"
+    # https://github.com/dnicolodi/python-siphash24/issues/8:
+    - "PYTHONHASHSEED=0 python check.py"  # [not ((ppc64le or aarch64) and py<312)]
```

Comment:
How come ppc64le and aarch64 get a pass?

Reply:
I suspect their older python builds (<3.12) use FNV hashes by default on conda-forge builds, which is why it fails due to https://github.com/dnicolodi/python-siphash24/issues/8. See also https://peps.python.org/pep-0456/. Why this gets triggered in this PR I don't know, but it may be due to switching to cross-compilation for these 2 arches.

Reply:
Thanks for explaining!

Reply:
How come your arm64 Python does not use SipHash hashing? It is rather strange.

Reply:
It seems to be a bug in conda-forge's python package: https://github.com/conda-forge/python-feedstock/issues/718 https://github.com/conda-forge/python-feedstock/pull/719. Like I speculated above, it seems to happen only when cross-compiling python.

Reply:
Then it is a bug in how the cross compilation is setup or in the autoconf used to detect the requirement for aligned reads in the Python build.